include/oneapi/dpl/internal/distributed_ranges_impl/detail/segments_tools.hpp

```diff
@@ -16,7 +16,7 @@
 #ifndef _ONEDPL_DR_DETAIL_SEGMENT_TOOLS_HPP
 #define _ONEDPL_DR_DETAIL_SEGMENT_TOOLS_HPP
 
-#include "enumerate.hpp"
+#include "../sp/views/enumerate.hpp"
```

Comment:
we use only stdrng::enumerate now so including ../sp/views/enumerate.hpp is not needed